hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieFileReaderFactory.java

```diff
@@ -46,13 +44,11 @@
     throw new UnsupportedOperationException(extension + " format not supported yet.");
   }
 
-  private static <T extends HoodieRecordPayload, R extends IndexedRecord> HoodieFileReader<R> newParquetFileReader(
-      Configuration conf, Path path) throws IOException {
```

Comment:
this method never throws IOException. Can you remove that as well?

Reply:
It is already removed from `newParquetFileReader`.